### PR TITLE
op-guide: update test and production suggestion in binary deployment.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@queenypingcap @shenli @iamxy @siddontang  PTAL

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: op-guide/binary-deployment.md [hunk 1]
```diff
@@ -29,7 +27,7 @@ tar -xzf tidb-latest-linux-amd64.tar.gz
 cd tidb-latest-linux-amd64
 ```
 
-#### CentOS 6
+#### CentOS 6 (Not Suggested)
```
Comment (queenypingcap) [inline op-guide/binary-deployment.md]: Suggested -> Recommended

Comment (qiuyesuifeng) [inline op-guide/binary-deployment.md]: Got it.

**Comment by queenypingcap** [comment]
LGTM

**Pull request merged**